### PR TITLE
Implemented Opset 13 for the Sign, Sqrt, and Tanh  operators.

### PR DESCRIPTION
**Pull request by shahirdaya**
The corresponding TF operators supported bfloat16 and I tested them with bfloat16 tensors. Hence no additional tests were required.
The only change from the previously implemented Opset to Opset 13 was support for bfloat16.

### CODE REVIEW

**Review by winnietsang — CHANGES_REQUESTED** [review]

File: onnx_tf/handlers/backend/sign.py [hunk 1]
```diff
@@ -13,3 +13,7 @@ class Sign(BasicMathMixin, BackendHandler):
   @classmethod
```
[the remark this answers is not in the document]
Comment (shahirdaya) [reply onnx_tf/handlers/backend/sign.py:13]: @winnietsang thanks for the feedback. I'll address this. Thanks

[the remark this answers is not in the document]
Comment (shahirdaya) [reply onnx_tf/handlers/backend/sign.py:52]: Done.  Please. review and let me know @winnietsang. Thanks

File: onnx_tf/handlers/backend/sign.py [hunk 2]
```diff
@@ -4,12 +4,55 @@
...
 @onnx_op("Sign")
 @tf_func(tf.sign)
 class Sign(BasicMathMixin, BackendHandler):
+  cast_map = {
+    tf.uint8: tf.int32,
+    tf.uint16: tf.int32,
+    tf.uint32: tf.int64,
+    tf.int8: tf.int32,
+    tf.int16: tf.int32,
+    tf.float16: tf.float32
```
Comment (winnietsang) [inline onnx_tf/handlers/backend/sign.py:21]: tf.float16 is a supported dtype for tf.sign, no need to cast it to tf.float32

File: onnx_tf/handlers/backend/sign.py [hunk 3]
```diff
@@ -4,12 +4,55 @@
...
+          data_type.tf_to_np_str_list(cls.supported_types))
+
+  @classmethod
+  def _common(cls, node, **kwargs):
+    x = kwargs["tensor_dict"][node.inputs[0]]
+
+    # handle data types that are not natively supported by Tensorflow
+    dtype = x.dtype
+    x = tf.cast(x, cls.cast_map[dtype]) if dtype in cls.cast_map else x
+    inputs = [x]
```
Comment (winnietsang) [inline onnx_tf/handlers/backend/sign.py:47]: why not directly put this into the next statement.

**Review by winnietsang — CHANGES_REQUESTED** [review]
@shahirdaya May you add a testcase for sign of one of those unsupported datatype to verify the data casting is working please.

File: onnx_tf/handlers/backend/sign.py (hunk 2 above)
Comment (shahirdaya) [reply onnx_tf/handlers/backend/sign.py]: I have addressed this. Thanks for catching it.

File: onnx_tf/handlers/backend/sign.py (hunk 3 above)
Comment (shahirdaya) [reply onnx_tf/handlers/backend/sign.py]: Corrected this. Thanks

**Comment by shahirdaya** [comment]
I have added a test to drive the data cast code path. Thanks

**Review by winnietsang — APPROVED** [review]
LGTM

**Pull request merged**